### ISSUE TITLE
Issue: Need help with Navigation link text styles

### ISSUE THREAD

**Issue by mmenestr** [patternfly/patternfly-design-kit#40]
@mceledonia @kybaker 

For context, I have been working on the Navigation components, and have found a lot of font usages that I'm unclear about or didn't exist. For example, the current light link color text option is in regular weight, but in the horizontal nav component it's actually in semi-bold, so I decided to create a new text style for that.

When I was previously working on Action buttons, I had seen that the 16px links were all in semi-bold, and determined that this was the case every-time, so I updated the pf-global-link--color to be semi-bold instead of regular (what it used to be). 

Now, upon further inspection, I noticed that the hover states for these selected links use regular weight font, while the default/selected states depending on the usage, are all semi-bold. **The issue is that there is no regular weight font style for this now (since I had changed it to semibold as it seems to be the new default link style), so I'm wondering whether I should make a new hover state for nav links now that uses a regular weight?** An example of this is in the screenshot below where the subnav link 1 is being hovered over and is in regular weight, but if I were to select it it would become semi-bold.

<img width="845" alt="Screen Shot 2019-06-26 at 1 59 07 PM" src="https://user-images.githubusercontent.com/51165119/60203714-3e673180-981b-11e9-94fb-b46a071b337b.png">

Another problem, is that as you can see in the screenshot above, although Link 2 and Link 4 are both selected, one is in regular font weight, and the other is in semibold weight. Is this just a glitch and should these all be semi-bolded?

**In summary, the main question is - is it OK to make a new font style that is regular for hovered nav links?** 

Sorry for the long explanation, let me know if you have any questions! Thank you!!

**Comment by gdoyle1** [comment]
Closing because discussed this during the summer
